src/RecurringGrantDrop.sol

```diff
@@ -146,7 +147,18 @@ contract RecurringGrantDrop is Ownable2Step {
 
     ///////////////////////////////////////////////////////////////////////////////
     ///                               CLAIM LOGIC                               ///
-    //////////////////////////////////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////////
+
+    /// @notice Block a nullifier hash from being used
+    /// @param nullifierHash The nullifier hash to block
+    /// @param signature The signature that ensures this function can only be called by an allowed nullifier hash blocker
+    function setNullifierHash(uint256 nullifierHash, bytes calldata signature) public {
+        address signer = ECDSA.recover(keccak256(abi.encode(nullifierHash)), signature);
```

Comment:
the reservation logic signs both a timestamp and a nullifierHash but I think for this function, the nullifierHash is sufficient?

Reply:
unless we change the logic of signature generation, the contract expects to recover:

`keccak256(abi.encode(timestamp, nullifierHash)), signature`